site/en/docs/handbook/test-post/index.md

```diff
@@ -58,8 +58,16 @@ alt: A description of the hero image for screen reader users.
 tags:
   - privacy
   - security
+pageScripts:
+  - /js/100.js
 ---
 
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dictum a massa
+sit amet ullamcorper. Suspendisse auctor ultrices ante, nec tempus nibh varius
+at. Cras ligula lacus, porta vitae maximus a, ultrices a mauris. Vestibulum
+
+<like-icon item='some-id'></like-icon>
```

Comment:
Can you put a few more test LikeIcons on this page, just to confirm how it behaves? (See my earlier comment about sharing the Firebase instance as a singleton.)

Reply:
I think initializeApp takes care of this scenario, but i move i out of the instance nonetheless.